### PR TITLE
Language and variables fix

### PR DESCRIPTION
**Pull request by lazmeister**
This is literally my first PR on an open source project in a long time. Hope this is the required information!

-- Changes --
- Updated comments with grammar/translations to English
- Updated variable _isAutostarting to _isAutoStarting
- Updated event emitter autostarted to autoStarted
- Added an argument null on line 611  `this.camerasNotFound.next(null)`
- Updated version to 3.4.1

### CODE REVIEW

**Comment by lazmeister** [comment]
I did start with updating RXJS but reversed it as I do not have enough knowledge yet to update 659, sorry!

**Comment by lazmeister** [comment]
Sorry I think I clicked merge too early (it's been a long time since last PR)

File: projects/zxing-scanner/src/lib/zxing-scanner.component.ts [hunk 1]
```diff
@@ -99,16 +99,16 @@ export class ZXingScannerComponent implements OnInit, OnDestroy {
   delayBetweenScanSuccess = 500;
 
   /**
-   * Emits when and if the scanner is autostarted.
+   * Emits when and if the scanner is autoStarted.
    */
   @Output()
-  autostarted: EventEmitter<void>;
```
Comment (werthdavid) [inline projects/zxing-scanner/src/lib/zxing-scanner.component.ts:105]: That's a breaking change (public API).. Even though it looks better with the uppercase "S" I suggest to revert it

**Comment by werthdavid** [comment]
Much appreciated, thanks!

File: projects/zxing-scanner/src/lib/zxing-scanner.component.ts (hunk 1 above)
Comment (lazmeister) [reply projects/zxing-scanner/src/lib/zxing-scanner.component.ts:105]: no problem, I'll push a change and revert the variable. Should have thought about that. Thanks!

**Comment by lazmeister** [comment]
Let me know if this is good. Pushed up the changes and reverted the variable

File: projects/zxing-scanner/src/lib/zxing-scanner.component.ts [hunk 2]
```diff
@@ -108,7 +108,7 @@ export class ZXingScannerComponent implements OnInit, OnDestroy {
    * True during autostart and false after. It will be null if won't autostart at all.
    */
   @Output()
-  autostarting: EventEmitter<boolean>;
+  autoStarting: EventEmitter<boolean>;
```
Comment (werthdavid) [inline projects/zxing-scanner/src/lib/zxing-scanner.component.ts:111]: Sorry, I think that is the Problem with the API. this should be "autostarting" again.

File: projects/zxing-scanner/src/lib/zxing-scanner.component.ts [hunk 3]
```diff
@@ -304,16 +304,16 @@ export class ZXingScannerComponent implements OnInit, OnDestroy {
   /**
    *
    */
-  set isAutostarting(state: boolean) {
-    this._isAutostarting = state;
-    this.autostarting.next(state);
+  set isAutoStarting(state: boolean) {
```
Comment (werthdavid) [inline projects/zxing-scanner/src/lib/zxing-scanner.component.ts:307]: Aalso this and the getter below

**Comment by lazmeister** [comment]
okay that should do it for putting them all back to the originals. Sorry about that!

File: projects/zxing-scanner/src/lib/zxing-scanner.component.ts [hunk 4]
```diff
@@ -195,11 +195,11 @@ export class ZXingScannerComponent implements OnInit, OnDestroy {
 
     if (!this._ready) {
       this._devicePreStart = device;
-      // let's ignore silently, users don't liek logs
+      // let's ignore silently, users don't like logs
       return;
     }
 
-    if (this.isAutostarting) {
+    if (this.iAutostarting) {
```
Comment (werthdavid) [inline projects/zxing-scanner/src/lib/zxing-scanner.component.ts:202]: there is an "s" missing.. "isAu.." and not "iAu"..
Sorry for all the inconvenience 😅

Comment (lazmeister) [reply projects/zxing-scanner/src/lib/zxing-scanner.component.ts]: ahhhhh! Sorry, I should have stashed my changes and started fresh. Okay, pushed up another.

**Review by werthdavid — APPROVED** [review]

**Pull request merged**